doc/mkdoc.sh

```diff
@@ -13,7 +13,8 @@ mkdir -p ${tmpdir}/figs
 cp -u ${rootdir}/doc/img/bigmap.png ${tmpdir}/figs/.
 
 ontograph -m ${rootdir}/electrochemistry.ttl ${tmpdir}/electrochemistry-structure.png
-ontoconvert -si -- ${rootdir}/electrochemistry.ttl ${tmpdir}/electrochemistry-inferred.ttl
+#ontoconvert -si -- ${rootdir}/electrochemistry.ttl ${tmpdir}/electrochemistry-inferred.ttl
```

Comment:
Is the ambition to enable reasoning again? What do we need it for? If that is the case, please add a comment saying so.
If that is not the case, we can remove the commented out line.